quickstarts/aws/aws-smd/config.yml

```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist/) to learn 
+  more about New Relic monitoring and integration with AWS System Manager Distributor.
+summary: |
+  Use New-Relic-infrastructure-monitoring-agent tile to install the agent and monitor your EC2 instances with New Relic
```

Comment:
The wording here is a little suspect, I don't think user's will understand what a "tile" is, I'm not quite sure myself. If it's referring to the "tile" in the IO catalogue page view, then we definitely don't want to refer to it that way. I'd like to get this updated before we publish it.